media-art-background.js

```diff
@@ -42,6 +42,8 @@ function setBackground(root, appLayout, lovelace, bgroundElem) {
     let entityName = entity.entity || entity;
     let entityValidStates = entity.valid_states || ['playing'];
     let entityImageAttribute = entity.image_attribute || 'entity_picture';
+    let entityValidViews = entity.views; //get a list of valid views
```

Comment:
Can you also document this new option in the README?

Reply:
Something like that?